install_cli.php

```diff
@@ -108,6 +126,9 @@
         $generatorName = '\Sugarcrm\Tidbit\Generator\\' . $module;
         /** @var \Sugarcrm\Tidbit\Generator\Common $generator */
         $generator = new $generatorName($GLOBALS['db'], $storageAdapter, $insertBatchSize);
+        if (!empty($GLOBALS['as_populate'])) {
```

Comment:
Please add ```\n\tWill create " . $activityGenerator->calculateActivitiesToCreate($total) . " activity records \n";``` so we have same output for modules that are using Generators and general modules